docker/panw-iot/Pipfile

```diff
@@ -8,6 +8,10 @@ websocket-client = "*"
 websockets = "*"
 oauthlib = "*"
 requests-oauthlib = "*"
+pandas = "*"
+pywinrm = "*"
+# For pysnmp and snap7 both to work we need this specific version.
+pyasn1 = {version="==0.4.8"}
```

Comment:
can we check if pipenv lock is able to catch this? If it can its better to leave the shared subdependency logic to pipenv and leave this with *